botorch/test_functions/multi_fidelity.py

```diff
@@ -139,3 +139,45 @@ def evaluate_true(self, X: Tensor) -> Tensor:
         t1 = 100 * (X_next - X_curr ** 2 + 0.1 * (1 - X[..., -2:-1])) ** 2
         t2 = (X_curr - 1 + 0.1 * (1 - X[..., -1:]) ** 2) ** 2
         return -((t1 + t2).sum(dim=-1))
+
+
+class MFForrester(SyntheticTestFunction):
```

Comment:
Let's put this in a separate PR, since this change is unrelated to the rest of the PR

Reply:
I can generate another PR for this when this current one has been merged.